mappings/markdown/Prowler/samples/prowler.ocsf

```diff
@@ -0,0 +1,143 @@
+{
+    "metadata": {
+        "event_code": "acm_certificates_expiration_check",
+        "product": {
+            "name": "Prowler",
+            "vendor_name": "Prowler",
+            "version": "4.4.0"
+        },
+        "version": "1.2.0"
+    },
+    "severity_id": 4,
+    "severity": "High",
+    "status": "Other",
+    "status_code": "PASS",
+    "status_detail": "ACM Certificate c5872ce2-4a85-4486-91bc-be28a01f9ad5 for *.example.com expires in 202 days.",
+    "status_id": 99,
+    "unmapped": {
+        "check_type": "Data Protection",
```

Comment:
finding_info.type?

Reply:
That'd be a great addition, if you think that makes sense we can add that there as
```json
...
"finding_info": {
    "type": "Data Protection",
    ...
}
...
```

Reply:
Done!